conductr_cli/host.py

```diff
@@ -43,4 +44,55 @@ def with_docker_machine():
 
 
 def loopback_device_name():
-    return 'lo' if platform.system().lower() == 'linux' else 'lo0'
+    return 'lo' if is_linux() else 'lo0'
+
+
+def is_linux():
+    return platform.system().lower() == 'linux'
+
+
+def is_osx():
```

Comment:
Nowadays it is called `macOS` instead of `osx`. So use `macos` here. We are using the term `macOS` already here: https://github.com/typesafehub/conductr-cli/blob/master/conductr_cli/sandbox_main.py#L164